2025-03-24 22:49:55,041 - INFO - Assets path configured: /home/ryan/main/python/assets
2025-03-24 22:49:55,042 - INFO - Starting up UDS server on /tmp/DFd0LmdPEG
2025-03-24 22:49:55,042 - INFO - Flet app has started...
2025-03-24 22:50:01,708 - INFO - Assets path configured: /home/ryan/main/python/assets
2025-03-24 22:50:01,709 - INFO - Starting up UDS server on /tmp/9C7ipEOh3g
2025-03-24 22:50:01,709 - INFO - Flet app has started...
2025-03-24 22:50:02,098 - INFO - App session started
2025-03-24 22:50:15,556 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'board_container'")>
Traceback (most recent call last):
  File "/usr/lib/python3.13/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/ryan/main/python/venv/lib/python3.13/site-packages/flet/core/page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "/home/ryan/main/python/app.py", line 358, in <lambda>
    on_click=lambda e, c=color["value"]: self.change_board_color(c)
                                         ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/home/ryan/main/python/app.py", line 374, in change_board_color
    self.page.board_container.bgcolor = color
    ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'board_container'
2025-03-24 22:50:16,085 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'board_container'")>
Traceback (most recent call last):
  File "/usr/lib/python3.13/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/ryan/main/python/venv/lib/python3.13/site-packages/flet/core/page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "/home/ryan/main/python/app.py", line 358, in <lambda>
    on_click=lambda e, c=color["value"]: self.change_board_color(c)
                                         ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/home/ryan/main/python/app.py", line 374, in change_board_color
    self.page.board_container.bgcolor = color
    ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'board_container'
2025-03-24 22:50:16,660 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'board_container'")>
Traceback (most recent call last):
  File "/usr/lib/python3.13/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/ryan/main/python/venv/lib/python3.13/site-packages/flet/core/page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "/home/ryan/main/python/app.py", line 358, in <lambda>
    on_click=lambda e, c=color["value"]: self.change_board_color(c)
                                         ~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/home/ryan/main/python/app.py", line 374, in change_board_color
    self.page.board_container.bgcolor = color
    ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'board_container'
2025-03-24 22:50:17,308 - ERROR - Future exception was never retrieved
future: <Future finished exception=AttributeError("'Page' object has no attribute 'tables'")>
Traceback (most recent call last):
  File "/usr/lib/python3.13/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/ryan/main/python/venv/lib/python3.13/site-packages/flet/core/page.py", line 903, in wrapper
    handler(*args)
    ~~~~~~~^^^^^^^
  File "/home/ryan/main/python/app.py", line 379, in reset_layout
    for i, table in enumerate(self.page.tables):
                              ^^^^^^^^^^^^^^^^
AttributeError: 'Page' object has no attribute 'tables'
